packages/cli/src/commands/network/parameters.ts

```diff
@@ -0,0 +1,106 @@
+import { BaseCommand } from '../../base'
+
+export default class Parameters extends BaseCommand {
+  static description = 'View network parameters'
+
+  static flags = {
+    ...BaseCommand.flags,
+  }
+
+  async run() {
+    const goldToken = await this.kit.contracts.getGoldToken()
+    const stableToken = await this.kit.contracts.getStableToken()
+    const attestations = await this.kit.contracts.getAttestations()
+    console.log(
+      'attestations.attestationExpirySeconds',
+      (await attestations.attestationExpirySeconds()).toString(10),
+      'seconds'
+    )
+    console.log(
+      'attestations.attestationExpiryFee (gold)',
+      (await attestations.attestationRequestFees(goldToken.address)).toString(10),
+      ''
+    )
+    console.log(
+      'attestations.attestationExpiryFee (stable)',
+      (await attestations.attestationRequestFees(stableToken.address)).toString(10),
+      ''
+    )
+
+    const lockedGold = await this.kit.contracts.getLockedGold()
+    console.log(
+      'lockedGold.maxNoticePeriod',
+      (await lockedGold.maxNoticePeriod()).toString(10),
+      'seconds'
+    )
+
+    const sortedOracles = await this.kit.contracts.getSortedOracles()
+    console.log(
+      'sortedOracles.reportExpirySeconds',
+      (await sortedOracles.reportExpirySeconds()).toString()
+    )
+
+    const exchange = await this.kit.contracts.getExchange()
+    console.log('exchange.spread', (await exchange.spread()).toString())
+    console.log('exchange.reserveFraction', (await exchange.reserveFraction()).toString())
+    console.log('exchange.updateFrequency', (await exchange.updateFrequency()).toString())
+    console.log('exchange.minimumReports', (await exchange.minimumReports()).toString())
+
+    const governance = await this.kit.contracts.getGovernance()
+    console.log(
+      'governance.concurrentProposals',
+      (await governance.concurrentProposals()).toString()
+    )
+    console.log('governance.dequeueFrequency', (await governance.dequeueFrequency()).toString())
+    console.log('governance.minDeposit', (await governance.minDeposit()).toString())
+    console.log('governance.queueExpiry', (await governance.queueExpiry()).toString())
+    const durations = await governance.stageDurations()
```

Comment:
Instead of parsing the output here. We should make `governance.stageDuraation()` return an object with `{approvalStage, referendumStage, executionStage}`. Use the postParse param from proxyCall

Reply:
and maybe call it `getStageDurations()`